bedrock/mozorg/urls.py

```diff
@@ -197,39 +197,17 @@
     page('MPL/2.0/differences', 'mozorg/mpl/2.0/differences.html'),
     page('MPL/2.0/permissive-code-into-mpl', 'mozorg/mpl/2.0/permissive-code-into-mpl.html'),
 
-
+    page('contribute', 'mozorg/contribute/index.html'),
+    url('^contribute/embed/$', views.contribute_embed,
+        name='mozorg.contribute_embed'),
     page('contribute/events', 'mozorg/contribute/events.html'),
     url('^contribute/friends/$', views.contribute_friends, name='mozorg.contribute.friends'),
+    page('contribute/signup', 'mozorg/contribute/signup.html'),
     page('contribute/stories', 'mozorg/contribute/stories.html'),
-    page('contribute/stories/ruben', 'mozorg/contribute/story-ruben.html'),
     page('contribute/stories/faye', 'mozorg/contribute/story-faye.html'),
-    page('contribute/stories/shreyas', 'mozorg/contribute/story-shreyas.html'),
     page('contribute/stories/michael', 'mozorg/contribute/story-michael.html'),
-
-    url('^contribute/$', views.contribute_index, name='mozorg.contribute'),
-    url('^contribute/event/$', views.contribute,
-        kwargs={'template': 'mozorg/contribute/index.html',
-                'return_to_form': True},
-        name='mozorg.contribute_event'),
-    url('^contribute/embed/$', views.contribute_embed,
-        name='mozorg.contribute_embed',
-        kwargs={'template': 'mozorg/contribute/contribute-embed.html',
-                'return_to_form': False}),
-
-    page('contribute/signup', 'mozorg/contribute/signup.html'),
-    url('^contribute/task/',
```

Comment:
I'm getting an error locally navigating to `/contribute/signup/`. I think we still need this URL resolved for the individual tasks.